deployment/network-operator/values.yaml

```diff
@@ -280,6 +280,8 @@ ofedDriver:
       # podSelector: "app=myapp"
       # specify the length of time in seconds to wait before giving up for workload to finish, zero means infinite
       # timeoutSeconds: 300
+  # OFED container will store its build artifacts between runs to avoid driver recompilation on container rerun
+  driverInventoryPath: "/opt/mofed-inventory"
```

Comment:
Do we want to enable this feature in the helm chart while the corresponding OFED changes are still in progress?
Or we can postpone merging this PR until before April release

Reply:
For me it looks quite safe to merge this change now.

Reply:
I think we should wait at least for after having a branch for January